### PR TITLE
Feature/remember selected tenancy

### PR DESCRIPTION
**Pull request by hackneydrew**
screenshot - 09_10_2018 11_52_45

Going back to the Caller Details page when on a call should display a list of tenants at the selected property. This will have no effect for anonymous callers.

### CODE REVIEW

File: ng-ncc-app/src/app/pages/identify/identify.component.html [hunk 1]
```diff
@@ -1,46 +1,50 @@
...
-            (selected)="addressSelected($event)"></app-address-search-results>
+        <div *ngIf="!existing_call">
+
+            <!-- Search for contact details. -->
+            <h2 class="govuk-heading-l">Caller details</h2>
+
+            <!-- Search box - see https://github.com/alphagov/govuk-design-system-backlog/issues/132 -->
+            <app-panel class="govuk-!-margin-bottom-5">
+                <form action="">
+                    <div class="govuk-form-group govuk-!-margin-bottom-2">
```
Comment (LBHRErra) [inline ng-ncc-app/src/app/pages/identify/identify.component.html]: Is it possible to apply the styling to the form element and remove the div?

Comment (hackneydrew) [reply ng-ncc-app/src/app/pages/identify/identify.component.html]: Do you mean `<div *ngIf="!existing_call">`? It controls the presence of more than one element. Alternatively the condition could be added to each of the things to hide.

Comment (LBHRErra) [reply ng-ncc-app/src/app/pages/identify/identify.component.html]: I was referring to the div that's parent of the form: `<div class="govuk-form-group govuk-!-margin-bottom-2">`

Comment (hackneydrew) [reply ng-ncc-app/src/app/pages/identify/identify.component.html]: Ah!

**Pull request merged**